site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1001,6 +1025,17 @@ components:
             "status":
                $ref: '#/components/schemas/Status'
 
+      TenantWithId:
+         type: object
+         additionalProperties: false
+         allOf:
+            - type: object
+              properties:
+                 "id":
+                    description: The identifier of the tenant.
+                    type: string
+            - $ref: '#/components/schemas/Tenant'
```

Comment:
can we please move this to below the definition of the Tenant object?

Reply:
I have moved it.